banzai/main.py

```diff
@@ -142,7 +154,9 @@ def reduce_experimental_frames_console():
     run_end_of_night_from_console([reduce_experimental_frames])
 
 
-def reduce_frames_one_by_one(stages_to_do, pipeline_context, image_types=['EXPOSE', 'STANDARD']):
+def reduce_frames_one_by_one(stages_to_do, pipeline_context, image_types=None):
```

Comment:
Can the default just be set to `image_types=['EXPOSE', 'STANDARD']`?

Reply:
See above.